build.gradle

```diff
@@ -27,8 +27,9 @@ allprojects {
 
 subprojects {
     group = 'org.partiql'
-    version = '0.2.0'
+    version = '0.2.1-SNAPSHOT'
```

Comment:
You can ignore this line as its also part of #30, which will be merged before this commit.